### PR TITLE
Review View in MovieDetailVC

### PR DESCRIPTION
**Pull request by lihter**
- Created `ReviewView` that is simpler than in Figma Design (because I couldn't find any API calls for discussion tab)
- It consists of `socialLabel` (hardcoded to "Social"), `titleLabel` (review title), `writtenByLabel`, `reviewLabel` (actual text of someone's review), and `profileImageView` (which is the profile picture of the reviewer)
- Added the `getReview(for movieId: Int)` function to presenter which returns a single review for a movie (even if there are multiple reviews, but for now I will keep it this way since it's not specified in design how to switch between reviews... In the future I can easily change the function so it returns an array of reviews, but first I need to know how to switch through the reviews in the app)
- Also I added `ReviewViewModel` which has `author` (string), `createdAt` (string), `content`(string) and `profileImagePath`(URL?)

simulator_screenshot_1DEA445F-AFB5-4CE0-BE7D-CE322FD12315

### CODE REVIEW

File: MovieApp/MovieApp/App/Features/MovieBrowser/Components/DetailScreen/ReviewView/ReviewView+Design.swift [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        profileImageView.kf.setImage(with: review.profileImagePath)
+        profileImageView.layer.masksToBounds = false
+        profileImageView.layer.cornerRadius = imageSize.width / 2
+        profileImageView.clipsToBounds = true
+    }
+    
+    func defineLayoutForViews() {
+        socialLabel.snp.makeConstraints {
+            $0.top.equalToSuperview()
+            $0.leading.equalToSuperview().offset(4 * offset)
```
Comment (marinzg) [inline MovieApp/MovieApp/App/Features/MovieBrowser/Components/DetailScreen/ReviewView/ReviewView+Design.swift:71]: Missing `trailing`

File: MovieApp/MovieApp/App/Features/MovieBrowser/Components/DetailScreen/ReviewView/ReviewView+Design.swift [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        }
+        
+        profileImageView.snp.makeConstraints {
+            $0.top.equalTo(socialLabel.snp.bottom).offset(5 * offset)
+            $0.leading.equalToSuperview().offset(4 * offset)
+            $0.width.equalTo(imageSize.width)
+            $0.height.equalTo(imageSize.height)
```
Comment (marinzg) [inline MovieApp/MovieApp/App/Features/MovieBrowser/Components/DetailScreen/ReviewView/ReviewView+Design.swift]: I think you have `$0.size`

File: MovieApp/MovieApp/App/Features/MovieBrowser/Components/DetailScreen/ReviewView/ReviewView+Design.swift [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+            $0.top.equalTo(titleLabel.snp.bottom).offset(3 * offset)
+            $0.leading.equalTo(profileImageView.snp.trailing).offset(4 * offset)
+            $0.trailing.equalToSuperview().inset(4 * offset)
+        }
+        
+        reviewLabel.snp.makeConstraints {
+            $0.top.equalTo(writtenByLabel.snp.bottom).offset(6 * offset)
+            $0.leading.equalToSuperview().offset(4 * offset)
+            $0.trailing.equalToSuperview().inset(4 * offset)
+            $0.height.lessThanOrEqualTo(290)
```
Comment (marinzg) [inline MovieApp/MovieApp/App/Features/MovieBrowser/Components/DetailScreen/ReviewView/ReviewView+Design.swift:98]: Missing `bottom` constraint.

File: MovieApp/MovieApp/App/Features/MovieBrowser/MoviesViewController/MovieDetail/MovieDetailViewController+Design.swift [hunk 4]
```diff
@@ -49,7 +57,7 @@ extension MovieDetailViewController: DesignProtocol {
     func styleViews() {
         view.backgroundColor = .white
         
-        scrollView.contentSize = CGSize(width: UIScreen.main.bounds.width, height: 1000)
+        scrollView.contentSize = CGSize(width: UIScreen.main.bounds.width, height: 1400)
```
Comment (marinzg) [inline MovieApp/MovieApp/App/Features/MovieBrowser/MoviesViewController/MovieDetail/MovieDetailViewController+Design.swift:56]: This shouldn't be set manually. I've commented previously how adding top, bottom and width constraints eliminate the need to force scrollview content size.

**Review by marinzg — CHANGES_REQUESTED** [review]

File: MovieApp/MovieApp/App/Features/MovieBrowser/MoviesViewController/MovieDetail/MovieDetailViewController+Design.swift (hunk 4 above)
Comment (lihter) [reply MovieApp/MovieApp/App/Features/MovieBrowser/MoviesViewController/MovieDetail/MovieDetailViewController+Design.swift:56]: Changed in PR #35

File: MovieApp/MovieApp/App/Features/MovieBrowser/Components/DetailScreen/ReviewView/ReviewView+Design.swift (hunk 1 above)
Comment (lihter) [reply MovieApp/MovieApp/App/Features/MovieBrowser/Components/DetailScreen/ReviewView/ReviewView+Design.swift:71]: [Here](https://github.com/lihter/Movie-App/pull/29/commits/f40d6bede21f11464130253b6a1042774abbcee9#diff-bc89732f73c46afaa89ae45b57517d1ec13a170fa2c9ea73980713a236355845R72)

File: MovieApp/MovieApp/App/Features/MovieBrowser/Components/DetailScreen/ReviewView/ReviewView+Design.swift (hunk 2 above)
Comment (lihter) [reply MovieApp/MovieApp/App/Features/MovieBrowser/Components/DetailScreen/ReviewView/ReviewView+Design.swift]: [Here](https://github.com/lihter/Movie-App/pull/29/commits/f40d6bede21f11464130253b6a1042774abbcee9#diff-bc89732f73c46afaa89ae45b57517d1ec13a170fa2c9ea73980713a236355845R78)

File: MovieApp/MovieApp/App/Features/MovieBrowser/Components/DetailScreen/ReviewView/ReviewView+Design.swift (hunk 3 above)
Comment (lihter) [reply MovieApp/MovieApp/App/Features/MovieBrowser/Components/DetailScreen/ReviewView/ReviewView+Design.swift:98]: [Here](https://github.com/lihter/Movie-App/pull/29/commits/f40d6bede21f11464130253b6a1042774abbcee9#diff-bc89732f73c46afaa89ae45b57517d1ec13a170fa2c9ea73980713a236355845R97)

**Review by marinzg — APPROVED** [review]

**Pull request merged**